.github/workflows/go.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v3
       with:
-        go-version: 1.19
+        go-version: 1.20
 
```

Comment:
Since we have differing behaviour in Go 1.20 and pre-1.20, we might want to have this be a matrix that runs over 1.19 and the latest version of Go

Reply:
Now, add matrix that runs over 1.19 and the latest version of Go.